skimage/feature/_hog.py

```diff
@@ -5,17 +5,17 @@
 from . import _hoghistogram
 
 
-def _hog_normalize_block(block, method, eps=1e-5):
+def _hog_normalize_block(block, method, eps):
     if method == 'L1':
         out = block / (np.sum(np.abs(block)) + eps)
     elif method == 'L1-sqrt':
         out = np.sqrt(block / (np.sum(np.abs(block)) + eps))
     elif method == 'L2':
-        out = block / np.sqrt(np.sqrt(np.sum(block ** 2)) ** 2 + eps ** 2)
+        out = block / np.sqrt(np.sum(block ** 2) + eps ** 2)
```

Comment:
@vnavkal no, you are right, they are equivalent. I decided to follow the formulas exactly, but this way it should be faster.